app/importers/clients/sftp_client.rb

```diff
@@ -36,6 +37,20 @@ def dir_entries(*args)
   end
 
   private
+
+  def target_file(remote_file_name)
+    if target_directory.present?
+      "../#{target_directory}/#{remote_file_name}"
```

Comment:
This seems like it adds more bits that go into making the file path:
- `tmp/data_download/` in the download file method
- the `..` in this method
- `remote_filename` in the config
- and the actual filename

I'm wondering if there's benefits to this as compared to writing full paths into each remote filenames?  If there are differences between districts, seems good to fully decouple them into config.  Alternately, we could have the "target directory" in config be an absolute path, and remove the `..` and path `/tmp/data_download` paths that are here in code so that it's just `target_directory/remote_filename`.

Reply:
Oooooh I really like the idea of writing full paths into the remote filenames!